RecoMuon/MuonIdentification/python/muons1stStep_cfi.py

```diff
@@ -77,7 +77,10 @@
                                           OverlapDTheta = cms.double(0.02), # 1.14 degrees
                                           ClusterDPhi   = cms.double(0.6), # 34 degrees
                                           ClusterDTheta = cms.double(0.02) # 1.14
-    )
+    ),
+
+    # tracker muon arbitration
+    arbitrateTrackerMuons = cms.bool(True)
```

Comment:
I guess we can be "proactive" with this PR and have this change essentially universal.
The cases that will find some issues in the following more broad validation can have this flag set to false where needed.